Source/Meadow.Units/Conversions/ApparentPowerConversions.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Meadow.Units.Conversions
+{
+    internal static class ApparentPowerConversions
+    {
+        public static double Convert(double value, ApparentPower.UnitType from, ApparentPower.UnitType to)
+        {
+            if (from == to)
+            {
+                return value;
+            }
+            return value * apparentPowerConversions[(int)to] / apparentPowerConversions[(int)from];
+        }
+
+        //must align to enum
+        private static readonly double[] apparentPowerConversions =
+        {
+            0.000000001,		//Gigavolt amphere
+            0.000001,			//Megavolt amphere
+		    0.001,				//Kilovolt amphere
```

Comment:
something is amiss with indentation formatting here.

Reply:
Yeah mix of tab and spaces, I don't have auto formatting enabled on my laptop. Fixed. And we should do a full pass on Units.